css/sections/hero.css

```
@@ -1,531 +1,501 @@
 .hero-am-heading {
-	text-wrap: balance;
-	text-align: center;
-	max-width: 55%;
-	font-size: var(--font-size-xl-heading);
-	padding: 0;
-	text-wrap: balance;
-	text-align: center;
-	font-weight: 700;
-	cursor: default;
-	line-height: var(--lh-xl);
+  text-wrap: balance;
```

Stylelint (17.2.0)

[error] Unexpected duplicate "text-wrap" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)